.storybook/aoeu-theme.js

```diff
@@ -1,10 +1,10 @@
 import { create } from "@storybook/theming/create";
-import AOEULogo from "../images/logos/aoeu-logo-vertical.svg"
+import AOEULogo from "../packages/logo-svgs/aoeu-logo-vertical.svg"
 
 export const theme = create({
   base: "light",
-  brandTitle: "AOEU",
-  brandUrl: "https://theartofeducation.github.io",
+  brandTitle: "The Art of Education University Web/UI Style Guide",
+  brandUrl: "/",
```

Comment:
Did you mean to remove the URL from this line?

Reply:
"/" indicates it should be the root of the URL you are currently at. This way it would work if you're viewing it at `https:/theartofeducation.github.io` or `http://localhost:6006`.